### DISCUSSION TITLE
蓝灯在中国无法使用

### DISCUSSION THREAD

**Discussion by EXLF** [getlantern/lantern-forum-cn#551] [General]
@Mike777Khoo 购买了一年，结果无法使用，官方没开启申请退款通道，麻烦帮忙退款下，支付收据如下。
账号：2760869618@qq.com
![d285033be2a334c67f283199fe563d1](https://github.com/getlantern/lantern-forum-cn/assets/42761270/567338b7-7ddc-4011-a0df-be3b53af652a)
![1717863773695](https://github.com/getlantern/lantern-forum-cn/assets/42761270/633f661b-54f8-4ffa-8879-32476ac4133d)

**Comment by EXLF** [comment]
@Mike777Khoo 我无法联系上官方邮箱留言。

**Reply by MaxMKMax** [reply]
@EXLF 嗯嗯. 好的. 我这里会帮您提交上去处理. 请给我们两周的时间来完成退款.
